makefiles/Makefile.common

```diff
@@ -30,7 +30,14 @@ OPS_LIB_OMPOFFLOAD=-lops_ompoffload
 OPS_LINK=
 CUDART=-lcudart
 
-include $(OPS_INSTALL_PATH)/../makefiles/Makefile.$(OPS_COMPILER)
+COMPILER_AVAILABLE := clang gnu intel icx intel_win mic pgi cray xl hip xlp8 aocc
```

Comment:
this doesn't seem very scalable - could we just check what makefiles exist and throw the error if it doesn't?

Reply:
Sure, update the logic to check file exist or not